eval-jam/package.json

```diff
@@ -17,7 +18,7 @@
     "@types/rimraf": "^3.0.0",
     "@types/semver": "^7.3.1",
     "@types/table": "^5.0.0",
-    "acorn": "7.1.1",
+    "acorn": "7.1.0",
```

Comment:
I think the previous Acorn version has a security issue with `7.1.1` being the first version that is no longer vulnerable, so I believe we need the updated version. https://github.com/amzn/merlin-on-demand-callgraph/security/dependabot/1

Reply:
I'll try with the updated version but I believe that caused a type error. I'll see if something else I did fixed that.